extra-requirements.txt

```diff
@@ -81,3 +82,4 @@ trimesh:                    cicd
 paddlepaddle==2.2.0:        cicd
 onnx:                       cicd
 onnxruntime:                cicd
+sgqlc:                      cicd, graphql
```

Comment:
Is it a good idea to introduce this tag, or should i revert this?

Reply:
Let's not add it for the moment, as long as ImportExtensions is properly handled, it is okey